src/test/librbd/cache/test_mock_ReplicatedWriteLog.cc

```diff
@@ -39,6 +38,9 @@ inline ImageCtx *get_image_ctx(MockImageCtx *image_ctx) {
 #include "librbd/cache/WriteLogCache.cc"
 #include "librbd/cache/pwl/AbstractWriteLog.cc"
 #include "librbd/cache/pwl/ReplicatedWriteLog.cc"
+#ifdef WITH_RBD_SSD_CACHE
+#include "librbd/cache/pwl/SSDWriteLog.cc"
```

Comment:
Nit: the name of the test is `test_mock_ReplicatedWriteLog` so does it really need the `SSDWriteLog.cc`? I would think you would create a new `test_mock_SSDWriteLog` set of test cases.

Reply:
Right, but this class should really just be testing `ReplicatedWriteLog` directly (not through the `WriteLogCache` wrapper).

Reply:
... and probably technically can move to `test/librbd/cache/pwl/...` directory / namespace now.

Reply:
Done